app/helpers/vulnerabilities_helper.rb

```diff
@@ -11,7 +11,7 @@ def major_releases(releases, project)
       releases.pluck(:version)
     else
       releases.map do |r|
-        r[:version].to_i
+        r[:version].gsub(/[^\d,.]/, '').to_i
```

Comment:
This logic will break when the pattern is `v.2.2.2`.
Change regex to `/^v?\.?/`. 
When changing regexp, always play around with it in irb with different inputs first.
Also add include this pattern in vulnerabilities_helper_test.